src/libraries/System.DirectoryServices/src/System/DirectoryServices/ActiveDirectory/ActiveDirectoryPartition.cs

```diff
@@ -6,9 +6,9 @@ namespace System.DirectoryServices.ActiveDirectory
     public abstract class ActiveDirectoryPartition : IDisposable
     {
         private bool _disposed;
-        internal string partitionName;
-        internal DirectoryContext context;
-        internal DirectoryEntryManager directoryEntryMgr;
+        internal string partitionName = null!;
+        internal DirectoryContext context = null!;
+        internal DirectoryEntryManager directoryEntryMgr = null!;
```

Comment:
Why are these ok?  Is it that all derived types always set them in the derived ctor?

Reply:
IIRC all derived classes filled these out and code was expecting them to be null. I'll double check

Reply:
yes, as I wrote above (minus corrected typo above)